crates/metrics/src/p2p_metrics.rs

```diff
@@ -28,3 +42,11 @@ static P2P_METRICS: OnceLock<P2PMetrics> = OnceLock::new();
 pub fn p2p_metrics() -> &'static P2PMetrics {
     P2P_METRICS.get_or_init(P2PMetrics::new)
 }
+
+pub fn inc_unique_peers() {
```

Comment:
Nit: I'd find this easier to read with a more descriptive name could we call this `increment_unique_peers`?